lib/runner/extensions/event.command.js

```diff
@@ -231,8 +231,8 @@ module.exports = {
                 // add event listener to trap all assertion events, but only if needed. to avoid needlessly accumulate
                 // stuff in memory.
                 (abortOnFailure || stopOnFailure) &&
-                    this.host.on(EXECUTION_ASSERTION_EVENT_BASE + executionId, function (cursor, assertion) {
-                        !assertion.passed && assertionFailed.push(assertion.name);
+                    this.host.on(EXECUTION_ASSERTION_EVENT_BASE + executionId, function (cursor, assertions) {
+                        !assertions[0].passed && assertionFailed.push(assertions[0].name);
```

Comment:
why checking only for `0`th item?

Reply:
Is there any dependency on`tests` object near to this code? If so, that has to be removed.